src/index.ts

```diff
@@ -30,10 +30,15 @@ export class DurableObject {
             // Reset the timeout with each ping by the client,
             // meaning it is still there and we shouldn't disconnect it.
             await ws.updateTimeout();
+
+            // The workaround here would be to trigger the DO's fetch() each time a listener gets triggered,
+            // but that would trigger one DO request for each message received.
+            await this.fetch(new Request('http://durable'));
```

Comment:
is the await needed?

Reply:
Actually, no.

This PR actually tries to solve nothing, it is more as a diff example of how this bug can be mitigated, but I've forwarded this repo to the Cloudflare DO team to fix it.